grails-app/services/au/org/ala/ecodata/ActivityService.groovy

```diff
@@ -367,4 +393,16 @@ class ActivityService {
         activities.collect{toMap(it, levelOfDetail)}
     }
 
+    def getAllActivityIdsForProjectActivity(String pActivityId) {
+        def criteria = Activity.createCriteria()
+        def list = criteria {
+            eq("projectActivityId", pActivityId)
+            eq("status", ACTIVE)
```

Comment:
Since we are deleting all activities for a Project Activity, it would be better to find ALL the activities, not just the ACTIVE ones.